File: src/B_10797/B_10797.java
package B_10797;

import java.io.*;
import java.util.*;

public class B_10797 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int c = Integer.parseInt(br.readLine());
        System.out.println(Arrays.stream(br.readLine().split(" ")).mapToInt(Integer::parseInt).filter(s -> s == c).toArray().length);

    }
}

File: src/B_10797/B_10797.kt
package B_10797

import java.io.*
import java.util.*

fun main() = with(BufferedReader(InputStreamReader(System.`in`))){
    val c: Int = readLine().toInt()
    print(Arrays.stream(readLine().split(" ".toRegex()).toTypedArray()).mapToInt { s: String -> s.toInt() }
        .filter { s: Int -> s == c }.toArray().size)
}